awx/ui_next/src/components/PromptDetail/PromptDetail.jsx

```diff
@@ -18,9 +18,19 @@ import PromptInventorySourceDetail from './PromptInventorySourceDetail';
 import PromptJobTemplateDetail from './PromptJobTemplateDetail';
 import PromptWFJobTemplateDetail from './PromptWFJobTemplateDetail';
 
-const PromptHeader = styled.h2`
-  font-weight: bold;
-  margin: var(--pf-global--spacer--lg) 0;
+const PromptTitle = styled(Title)`
+  margin-top: 40px;
```

Comment:
```suggestion
  margin-top: var(--pf-global--spacer--xl);
```
We could make them all PatternFly utility variables. Either way works for me.